src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -449,16 +449,21 @@ static void uct_ib_mlx5_devx_mr_lru_cleanup(uct_ib_mlx5_md_t *md)
     kh_destroy_inplace(rkeys, &md->lru_rkeys.hash);
 }
 
+static UCS_F_ALWAYS_INLINE int
+uct_ib_mlx5_devx_md_is_ksm_atomic_supported(uct_ib_mlx5_md_t *md)
+{
+    return ucs_test_all_flags(md->flags, UCT_IB_MLX5_MD_FLAG_KSM |
```

Comment:
To know that atomic KSM is supported we need to check to capabilities: `fixed_buffer_size` and `umr_modify_atomic_disabled`.
Now we have two flags for each one in MD.
Wouldn't it better instead to check both during init and have single flag for KSM_ATOMIC? 
This way we wan't need this helper.